resources/camino-trekker/components/MapMarkerLabel/MapMarkerLabel.vue

```diff
@@ -13,9 +13,11 @@
   </div>
 </template>
 <script setup lang="ts">
+export type MapMarkerLabelColors = "pink" | "orange" | "default";
+
 withDefaults(
   defineProps<{
-    color?: "pink" | "orange" | "default";
+    color?: MapMarkerLabelColors;
```

Comment:
I think you might be addicted to typescript 😂

Reply:
Totally – the autocompletion is addictive. 

Also, I don't think exporting types works from vue files, so I need to remove this.